library/unwind/build.rs

```diff
@@ -76,6 +88,16 @@ mod llvm_libunwind {
         let mut cpp_cfg = cc::Build::new();
         let root = Path::new("../../src/llvm-project/libunwind");
 
+        // We depend on starting with a fresh build directory each time.
```

Comment:
Hm, I'm wondering if we can get away with just deleting the directory as a whole here.

Reply:
Works on my machine; PTAL :)